### DISCUSSION TITLE
OpsAdministration>Core>PostInstallScript Jenkinsfile not found

### DISCUSSION THREAD

**Discussion by daveanvit** [Sunbird-Ed/Community#164] [Installation] [answered]
Can anyone please confirm my jenkins job configuration from below screenshot, is correct it correct as per the documentation and tag in job config they have different tag and jenkins file are different for those 
branch used for jobs import:- release-4.3.0

<img width="1440" alt="Screenshot 2022-09-15 at 10 32 12 AM" src="https://user-images.githubusercontent.com/107476510/190347050-8b5575fe-94a9-445b-9eba-4d3497446681.png">

**Comment by beepdot** [comment, marked as answer]
Please refer the the master branch for the configuration.
https://github.com/project-sunbird/sunbird-devops/blob/master/deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/Core/jobs/PostInstallScript/config.xml#L115
Please remove the hardcoded branch and use `${branch_or_tag}`
https://github.com/project-sunbird/sunbird-devops/blob/master/deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/Core/jobs/PostInstallScript/config.xml#L100
